domain/crossmodelrelation/types.go

```diff
@@ -178,7 +170,29 @@ type RemoteApplicationOfferer struct {
 // AddRemoteApplicationOffererArgs contains the parameters required to add a new
 // remote application offerer.
 type AddRemoteApplicationOffererArgs struct {
-	AddRemoteApplicationArgs
+	// ApplicationUUID is the UUID to assign to the synthetic application
+	// representing the remote application, on the consuming model.
+	ApplicationUUID string
+
+	// CharmUUID is the UUID to assign to the synthetic charm representing
+	// the remote application, on the consuming model.
+	CharmUUID string
+
+	// RemoteApplicationUUID is the UUID of the remote application, on the
+	// consuming model.
+	RemoteApplicationUUID string
```

Comment:
s/RemoteApplicationUUID/OfferingingApplicationUUID/

rename to be consistent with other recent naming changes to be explicit about which application this is.